test/test-run-appmetrics-usage.js

```diff
@@ -0,0 +1,51 @@
+// Copyright IBM Corp. 2015,2016. All Rights Reserved.
+// Node module: strong-supervisor
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+var debug = require('./debug');
+var run = require('./run-with-ctl-channel');
+var tap = require('tap');
+
+tap.test('appmetrics are forwarded via parentCtl', function(t) {
```

Comment:
If this is the test that needs the higher timeout, try adding `{timeout: 500*1000}` (the expanded `*1000` is to indicate to readers this is in ms, not s) as a 2nd arg to `tap.test`.